.github/CODEOWNERS

```diff
@@ -0,0 +1,18 @@
+# Temporary file to alert team members on the layout foundations project of changes to Card and Stack.
```

Comment:
I think this file was intentionally removed in v11 in a previous commit that merged main, so we could probably keep remove this.